.github/workflows/dapr.yml

```
@@ -27,7 +27,7 @@ jobs:
         uses: fkirc/skip-duplicate-actions@v3.4.0
         with:
           cancel_others: 'true'
-          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml"]'
+          paths_ignore: '["**.md", ".codecov.yaml", ".github/ISSUE_TEMPLATE/*"]'
```

Please add `.github/**/**.yml` again

```suggestion
          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml", ".github/ISSUE_TEMPLATE/*"]'
```